docs/MO_DG/prepare_model/Getting_performance_numbers.md

```diff
@@ -1,60 +1,57 @@
 # Getting Performance Numbers {#openvino_docs_MO_DG_Getting_Performance_Numbers}
```

Comment:
I think this one has been updated in https://github.com/openvinotoolkit/openvino/pull/11605. Can we copy all the changes here to avoid any conflicts?

Reply:
Changes have been copied. Thank you for pointing that out :)